docs/content/guides/developer/sui-101/using-events.mdx

```diff
@@ -36,159 +30,220 @@ To create an event in your Move modules, add the `sui::event` dependency.
+				"timestampMs": "1691757698019"
+			},
+			{
+				"id": {
+					"txDigest": "8b3byDuRojHXqmSz16PsyzfdXJEY5nZBGTM23gMsMAY8",
+					"eventSeq": "0"
+				},
+				"packageId": "0x158f2027f60c89bb91526d9bf08831d27f5a0fcb0f74e6698b9f0e1fb2be5d05",
+				"transactionModule": "deepbook_utils",
+				"sender": "0x8b35e67a519fffa11a9c74f169228ff1aa085f3a3d57710af08baab8c02211b9",
+				"type": "0xdee9::clob_v2::OrderFilled<0x2::sui::SUI, 0x5d4b302506645c37ff133b98c4b50a5ae14841659738d6d733d59d0d217a93bf::coin::COIN>",
+				"parsedJson": {
+					"base_asset_quantity_filled": "700000000",
+					"base_asset_quantity_remaining": "0",
+					"is_bid": false,
+					"maker_address": "0x03b86e93d80b27763ee1fc2c37e285465dff835769de9462d9ad4ebcf46ac6df",
+					"maker_client_order_id": "20082022",
+					"maker_rebates": "634",
+					"order_id": "9223372036854839643",
+					"original_quantity": "1000000000",
+					"pool_id": "0x7f526b1263c4b91b43c9e646419b5696f424de28dda3c1e6658cc0a54558baa7",
+					"price": "604100",
+					"taker_address": "0x704c8c0d8052be7b5ca7174222a8980fb2ad3cd640f4482f931deb6436902627",
+					"taker_client_order_id": "20082022",
+					"taker_commission": "1058"
+				},
+				"bcs": "DcVGz85dWTLU4S33N7VYrhgbkm79ENhHVnp5kBfENEWEjN45pa9U3AkNhxfTRZbaHTQLugLBXttE32hpJKRsbrZGdryXMPmNA8EpHJnVcnYMXZmWXkNXvY1XjEYnAKU4BnhyJ9BQuxRJDXLA4DEu5uWEpWjLPD2ZHuxqHCn7GpUxvxJjHkKjr9jVVfeR6sN2uRhUXkThEDjCekrqaqwidkyXNmTzmZG4fre3eoZ",
+				"timestampMs": "1691758372427"
+			}
+		],
+		"nextCursor": {
+			"txDigest": "8b3byDuRojHXqmSz16PsyzfdXJEY5nZBGTM23gMsMAY8",
+			"eventSeq": "0"
+		},
+		"hasNextPage": true
+	},
+	"id": 1
 }
 ```
+</details>
 
-## Subscribe to events in Move
+Both the TypeScript and Rust SDKs provide wrappers for the `suix_queryEvents` method.
 
-Firing events is not very useful in a vacuum. You also need the ability to listen for those events so that you can act on them. In Sui, you subscribe to those events and provide logic that triggers when the event fires.
+- TypeScript SDK [`client.queryEvents`](https://sdk.mystenlabs.com/typedoc/classes/_mysten_sui.client.SuiClient.html#queryEvents).
+- Rust SDK [`query_events`](https://mystenlabs.github.io/sui/sui_sdk/apis/struct.EventApi.html#method.query_events).
 
-Sui Full nodes support subscribe functionality using JSON-RPC notifications transmitted through the WebSocket API. You can interact with the RPC directly ([suix_subscribeEvent](/sui-api-ref#suix_subscribeEvent), [suix_subscribeTransaction](/sui-api-ref#suix_subscribeTransaction)) or you can use an SDK like the Sui TypeScript SDK. The following excerpt from one of the examples uses the TypeScript SDK to create an asynchronous subscription to the filter identified in the filter.
+### Query events with GraphQL
```

Comment:
This section seems like it's in the middle of two sections related to JSON-RPC (the previous example is JSON-RPC and the following section describes filters supported by `suix_queryEvents`), maybe it should go after?

Reply:
I thought the filters were the same...durp.